openedx/core/lib/xblock_builtin/xblock_discussion/xblock_discussion/__init__.py

```diff
@@ -2,8 +2,9 @@
 """
 Discussion XBlock
 """
+from __future__ import absolute_import
 import logging
-import urllib
+import six.moves.urllib.request, six.moves.urllib.parse, six.moves.urllib.error
```

Comment:
python-modernize imported all these because it wasn't sure how much was being used from urllib, but actually just the urlencode function was needed.  If it's not too much hassle, could you update this line to be just `from six.moves.urllib.parse import urlencode`, and update line 179 below to just be `qs = urlencode({`?